README.md

```diff
@@ -8,86 +8,82 @@ This list aims to be a curated set of high quality educational resources. The av
 ## Beginner
 
 #### [Codecademy](https://www.codecademy.com/)
-**Free** platform for learning to code in web technologies\
-*(HTML, CSS, JavaScript, jQuery, Python, Ruby, Rails, PHP)*
+**Freemium** Interactive platform for learning to code in many different programming languages\
+*(HTML, CSS, JavaScript, jQuery, Python, Ruby, Rails, PHP, C++, Java)*
 
 #### [Khan Academy's Hour of Code](https://www.khanacademy.org/hourofcode)
-**Free** interactive 1-hour course to learn the very basics of web development\
+**Free** Interactive one-hour courses to learn the very basics of web development.\
```

Comment:
- the style right now is a lowercase starting letter and no period
- for now, I'll remove the smaller changes like 1 -> one from this PR and possibly reconsider in future

```suggestion
**Free** interactive 1-hour courses to learn the very basics of web development\
```

Reply:
Where would I plug the code into Ann's how would I access it for my iphone